modules/python/src2/typing_stubs_generation/generation.py

```diff
@@ -314,6 +317,10 @@ def write_constant_to_stream(export_name: str) -> None:
     export_name = extra_export_prefix + constant_node.export_name
     write_constant_to_stream(export_name)
     if generate_uppercase_version:
+        # Handle Python "magic" constants like __version__
+        if re.match(r"^__.*__$", export_name) is not None:
+            return export_name,
```

Comment:
`,` is redundant

Reply:
Removing trailing `,` violates return type `Tuple[str, ...]`, thus requires an extra branch when return value is used:
```python
 generated_constants_entries.extend(
    _generate_constant_stub(entry, output_stream, indent, entries_extra_prefix)
)
```